cico_run_tests.sh

```diff
@@ -1,12 +1,11 @@
 
 #!/bin/bash
 
-. cico_setup.sh
+. ./cico_setup.sh
 
-TAG="test"
-TEST_IMAGE_NAME="$(make get-image-name):$TAG"
+TEST_IMAGE_NAME="$(make get-test-image-name)"
 
-./run_tests.sh
+. ./run_tests.sh
```

Comment:
Why this change is required?

Reply:
Same reason why I changed to . ./cico_setup.sh .

Reply:
To keep it on the line with   >". ./cico_setup.sh"  above on line 4.